### ISSUE TITLE
min and allowempylist does not work with TabularInput

### ISSUE THREAD

**Issue by michaelnguyen547** [unclead/yii2-multiple-input#152]
I am struggling to make TabularInput work with min and allowEmpyList

```
<?=
TabularInput::widget([
	'models' => [new TrainingCertificate()],
	'min' => 0,
	'allowEmptyList' => true,
	'addButtonOptions' => [
		'class' => 'btn btn-success',
		'label' =>  Html::tag('i', null, ['class' => 'glyphicon glyphicon-plus']) // also you can use html code
	],
	'attributeOptions' => [
		'enableAjaxValidation'      => true,
		'enableClientValidation'    => false,
		'validateOnChange'          => false,
		'validateOnSubmit'          => true,
		'validateOnBlur'            => false,
	],
	'columns' => $columns
])
?>
```

I still see 1 row rendered by the above code. I figure that because 

`'models' => [new TraingCertificate()]`

However, if I change models to something else

`'models' =>  null`

I get an error saying 

```
Invalid Configuration – yii\base\InvalidConfigException

You must specify "models"
```

**Comment by unclead** [comment]
Yes it is a bug and if you found out how to fix it - pr is always welcome.
i'm too busy now to fix it, mb i'll be able to fix it in a week

**Comment by ered15** [comment]
any fix for this issue?

**Comment by unclead** [comment]
no. if you need it you can fix it and send a PR

**Comment by unclead** [comment]
closed via be9194b0430fd9bcc4c0eae97de73b235b33249f
